ui-tests/src/test/java/org/openmrs/reference/LoginTest.java

```diff
@@ -33,46 +36,45 @@ public void verifyModulesAvailableOnHomePage() throws Exception {
         assertTrue(homePage.isDataManagementAppPresent());
         assertTrue(homePage.isConfigureMetadataAppPresent());
         assertTrue(homePage.isSystemAdministrationAppPresent());
-    }
-
+    } 
+    
```

Comment:
It should be an Enter key not an enter with spaces

Reply:
Resolved thanks